Bug in lidar2image_ndx generation for val queries

I've tried to implement validation during training and faced a problem that in the val phase the script returns an error:

> `AssertionError: Unknown lidar timestamp: 1435937763823973`

I've checked the index generation script and found that there is a bug in the code: https://github.com/jac99/MinkLocMultimodal/blob/e8fc6d23c3c6c3cb937186fd0efeb2b67a67b636/scripts/generate_rgb_for_lidar.py#L76

I am pretty sure that it should be `ts, traversal = get_ts_traversal(val_queries[e].rel_scan_filepath)`

--- a/scripts/generate_rgb_for_lidar.py
+++ b/scripts/generate_rgb_for_lidar.py
@@ -73,7 +73,7 @@ def index_scans_in_datasets(params: MinkLocParams):
         print('Indexing LiDAR scans in dataset: {} ...'.format(params.val_file))
         val_queries = get_queries(params.dataset_folder, params.val_file)
         for e in val_queries:
-            ts, traversal = get_ts_traversal(train_queries[e].rel_scan_filepath)
+            ts, traversal = get_ts_traversal(val_queries[e].rel_scan_filepath)
             scans_ts_ndx[ts] = traversal
 
     # Evaluation datasets

